docs/docs/tutorials/orchestration.md

```diff
@@ -3,7 +3,7 @@
 In this tutorial, you will learn how to set up and deploy the minimal Qlik Core stack
 comprising the core services. The core services represent the foundation on which to build full Qlik Core solutions.
 
-The core Qlik Core stack consists of the following services:
+The core stack of Qlik Core consists of the following services:
```

Comment:
Maybe we should say something like "The central stack of Qlik Core..."

Since the double "core" with different usages looks a bit weird to me? Thoughts?